src/modules/utils/marketplace.ts

```diff
@@ -1,3 +1,11 @@
+import { AhListing } from '@holaplex/marketplace-js-sdk';
+import { AhListingMultiMarketplace } from '../../pages/nfts/[address]';
+import {
+  AUCTION_HOUSE_ADDRESSES,
+  AUCTION_HOUSE_INFO,
+  MARKETPLACE_PROGRAMS,
+} from '../../views/_global/holaplexConstants';
```

Comment:
The constants should be in a marketplaces module and not pulled from views. The import direction should be from modules into views but not the reverse.